### PR TITLE
skip assert when a model returns tensor

### PR DESCRIPTION
**Pull request by yuyu2172**
The assertion triggers false positively.

This happens when I use a loss function that returns tensor (not tuple, dict, ...).

### CODE REVIEW

File: pytorch_pfn_extras/handler.py [hunk 1]
```diff
@@ -584,7 +584,8 @@ def train_step(self, models, optimizers, batch_idx, batch):
         to_back_outs = outs
         if self._grad_scaler is not None:
             to_back_outs = self._normalize_outputs(outs)
-            assert len(outs) == 1, "loss scaling with multiple outputs is not supported"
+            if not isinstance(outs, torch.Tensor):
+                assert len(outs) == 1, "loss scaling with multiple outputs is not supported"
```
Comment (emcastillo) [inline pytorch_pfn_extras/handler.py:588]:
```suggestion
             assert len(to_back_outs) == 1, "loss scaling with multiple outputs is not supported"
```

I think that `_normalize_outputs` will create a dict with the tensor, so this should be enough

**Review by emcastillo — APPROVED** [review]
